client/dashboard/domains/name-servers/index.tsx

```diff
@@ -56,10 +58,12 @@ export default function NameServers() {
 				<CardBody>
 					<NameServersForm
 						domainName={ domainName }
+						domainSiteSlug={ domain.site_slug }
```

Comment:
What's the difference between this and `getDomainSiteSlug()` 😄 cc: @arthur791004 

https://github.com/Automattic/wp-calypso/blob/c5e64fbcc29b5130ddb2398fab45b64d419b62a5/client/dashboard/utils/domain.ts#L12-L14

Reply:
Hey @hambai,

Any idea if there is a difference between the `domain.site_slug` field and the `getDomainSiteSlug` function?

Reply:
The `domain.site_slug` is empty for the primary domain

Reply:
Mind=blown

Reply:
Thank you, folks!
I replaced it.